dist/live-replica.js

```diff
@@ -82,7 +82,7 @@ var LiveReplica =
 /******/
 /******/
 /******/ 	// Load entry module and return exports
-/******/ 	return __webpack_require__(__webpack_require__.s = 10);
+/******/ 	return __webpack_require__(__webpack_require__.s = 11);
```

Comment:
This file is generated for distribution and contains the same changes found on the sources,
it should be ignored by the reviewer